src/coveralls.jl

```diff
@@ -11,7 +11,6 @@ module Coveralls
     using HTTP
     using JSON
     using Compat
-    using Compat.LibGit2
```

Comment:
Why was this removed? I'm fairly certain this is still needed.

Reply:
Is `LibGit2` not part of `Base`? (https://docs.julialang.org/en/stable/devdocs/libgit2/index.html)

Reply:
It is in Julia 0.6 but not in 0.7. In 0.6, `using Compat.LibGit2` is the same as `using Base.LibGit2`, and in 0.7 it's the same as `using LibGit2`, which is part of the standard library rather than Base. Either way, this is still required.

Reply:
I see ... Thanks for clarifying! I will add it back in ...